gpage.h

```diff
@@ -222,16 +235,19 @@ namespace gcpp {
 	//
 	inline
 	bool gpage::contains(gsl::not_null<const byte*> p) const noexcept {
-		return &storage[0] <= p && p < &storage[total_size - 1];
+		//  Use std::less<> to compare (possibly unrelated) pointers portably
+		auto const cmp = std::less<>{};
+		auto const ext = extent();
+		return !cmp(p, ext.data()) && cmp(p, ext.data() + ext.size());
```

Comment:
> submitted LWG 2562

Which I should point out was moved to Tentatively Ready at the LWG-only meeting in Chicago in August and will therefore likely be moved into the C++17 WP by NB comment along with all other Tentatively Ready LWG issue resolutions.